bin/sumd

```diff
@@ -52,6 +52,11 @@ parser.add_option("--config-dir", dest="config_directory", default="/etc/sumd",
 parser.add_option("--run-dir", dest="run_directory", default="/var/run", help="Directory for where pidfiles should be placed")
 parser.add_option("--foreground", dest="foreground", action='store_true', default=False, help="Don't daemonize.")
 parser.add_option("--debug", dest="debug", action='store_true', default=False, help="Increase logger verbosity")
+parser.add_option("--use-ssl", dest="use_ssl", action='store_true', default=False, help="Use SSL")
+parser.add_option("--keyfile", dest="keyfile", default="/etc/ssl/client.pkcs8", help="Path to SSL key file")
```

Comment:
Would it be easier for folks to understand if this was something like `--client-key-file` or something similar?